### ISSUE TITLE
User inteface doesn't come with option for saving Fit Width cropping option

### ISSUE THREAD

**Issue by cacygitmajonez** [tamius-han/ultrawidify#258]
I only managed to save Stretching option to Hybrid in deafult options in extension settings, but I couldn't find way to save Fit Width. I'm using super ultrawide monitor 32:9 ratio so please keep in mind users like me :) best regards

**Comment by tamius-han** [comment]
Yeah, 5.x version of the addon didn't support setting default crop mode.

Freshly released 6.x does have this option ~~except 6.0.0.1, which doesn't count as 6.x~~, but is currently only available on chrome until most of the major issues are found and fixed. Firefox and Edge are getting 6.0 _soon (tm)_.

**Comment by cacygitmajonez** [comment]
wow thanks :)

czw., 9 sty 2025 o 00:01 Tamius Han ***@***.***> napisał(a):
